rust/foxglove/README-development.md

```diff
@@ -0,0 +1,17 @@
+# Foxglove Rust SDK
```

Comment:
suggest renaming this file to `CONTRIBUTING.md` to match what we've done elsewhere.

or add it to the repo-level CONTRIBUTING.md